docs/dashboard/apps.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Apps
+---
+
+This page gives you an overview of how to manage apps in the dashboard.
+
+You can find the _Apps_ section on the main page of the dashboard. The Apps page allows you to install and manage your apps.
+
+The main page of Apps contains the following sections:
+
+- _Third-Party Apps_ a list of installed third-party apps in your dashboard.
+
+- _Local Apps_ with an option to create a new app.
+
+- _Saleor Marketplace_ - coming in Q3/Q4 2020.
```

Comment:
Q3/Q4? Are we sure? The priorities for marketplace has been changed and I don't think that we have enough resources to release marketplace in Q4. @patrys ?

Reply:
@patrys @korycins Maybe: release date - TBA?

Reply:
I'm not sure it's useful to mention a service that is not publicly available yet. Third-party apps do not have to come from the Saleor Marketplace. You can install from any site that publishes a valid App manifest file.

Reply:
@patrys So should we remove `Saleor Marketplace - Marketplace is coming soon` from the dashboard as well?

Reply:
I would remove any button that does not work if you're asking me. The marketplace will never arrive for 2.11 users as it's scheduled to launch after 3.x.

Reply:
I've removed that sentence about Saleor Marketplace from the docs for now.